vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -172,6 +171,16 @@ enum ExcludeFilter {
   FILTER_ALL_EXCEPT_EXCLUDED = 3
 };
 
+struct ActivityInfo {
```

Comment:
based on the fact that this is now split from `PublisherInfo`,  we should check what is actually needed in `PublisherInfo` and remove everything else. Please log an issue for it

Reply:
This was obsolete code, removed.